### ISSUE TITLE
Listing view - Order quantity field is not in the right navigation order

### ISSUE THREAD

**Issue by yusufttur** [shopware/platform#8554]
### Shopware Version

6.7.0.0

### Affected area / extension

Extension:B2B-Suite

### Actual behaviour

The order quantity field in the list view is skipped and highlighted after finishing all navigation points on that page.

https://github.com/user-attachments/assets/b4c20477-79b0-484b-bf82-af9e35f0dbd8

### Expected behaviour

It should be highlighted after product number

### How to reproduce

Keyboard navigation
